src/model/task.ts

```diff
@@ -1,11 +1,19 @@
 import { model, Schema } from 'mongoose'
 
+export enum TaskImportance {
+  LOW ='LOW', MEDIUM = 'MEDIUM', HIGH = 'HIGH'
```

Comment:
Please, do not write enums in one line. Better to write one enum value per line

Reply:
30afaebf219a08e95436d413662504fff035dd68